from .exception_base import ExceptionBase


class ArgumentException(ExceptionBase):
    pass
